apps/server/src/schema/organization/organization-operations.ts

```diff
@@ -359,6 +359,22 @@ builder.mutationFields((t) => ({
       });
     },
   }),
+
+  switchTiers: t.withAuth({ isAdmin: true }).prismaField({
+    type: OrganizationDto,
+    nullable: false,
+    args: {
+      organizationId: t.arg.string({ required: true }),
+      tier: t.arg({ type: Tier, required: true }),
+    },
+    resolve: async (query, _root, args, _ctx, _info) => {
+      const { organizationId, tier: newTier } = args;
+
+      const updatedOrganization = switchTierHelper(organizationId, newTier, query);
+
+      return updatedOrganization;
```

Comment:
```suggestion
      return switchTierHelper(organizationId, newTier, query);
```
let's try to return immediately when possible.